applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -230,29 +230,29 @@ config BLE_ISO_RX_STATS_S
 	int "Interval in seconds to print Bluetooth LE ISO RX stats. 0 to deactivate"
 	default 0
 
-choice BT_AUDIO_UNICAST_BAP_CONFIGURATION
+choice BT_BAP_UNICAST_BAP_CONFIGURATION
```

Comment:
Hmm, I think maybe the best naming scheme could be:
```suggestion
choice LE_AUDIO_UNICAST_BAP_CONFIGURATION
```
Our .c/.h files are called le_audio_(...)

Reply:
In other words: BT_AUDIO -> LE_AUDIO

Reply:
What is the reasoning for keeping the BAP name scheme?

Reply:
I believe that is to match the renaming in host. We should have a separate task for changing all our configs to LE_AUDIO_... perhaps?

Reply:
Yes, we can discuss this in the architecture discussions